### ISSUE TITLE
changing header color

### ISSUE THREAD

**Issue by cordovapolymer** [Taracque/cordova-plugin-braintree#17]
How can the blue header color be changed?
Do I have to modify braintree sdk itself?

**Comment by Taracque** [comment]
Where you see that blue header?

**Comment by cordovapolymer** [comment]
@Taracque confused you, it's gray color header.

**Comment by Taracque** [comment]
Can you please tell where is that header? Is it a PayPal thing, is it on iOS or Android?

**Comment by cordovapolymer** [comment]
it's in credit card entry screen on Android

**Comment by Taracque** [comment]
You mean this one?
![client-sdk-android-drop-in-series](https://cloud.githubusercontent.com/assets/373777/24956802/a29040d2-1f89-11e7-8413-29739f1534b7.png)
This one is part of the Braintree Drop-IN ui, there is no easy way to change that.

**Comment by cordovapolymer** [comment]
yes, as far as I have looked into this, there's a [way to specify a theme](https://github.com/braintree/braintree_android/issues/98#issuecomment-220184695) for BraintreePaymentActivity. 
In this case the activity has another name com.braintreepayments.api.BraintreeBrowserSwitchActivity.

**Comment by cordovapolymer** [comment]
@Taracque any thoughts on this?

**Comment by Taracque** [comment]
I guess the activity name is AddCardActivity in the com.braintreepayments.api.dropin package.
You should try to add the following to the manifest to style that window:
```
        <activity android:name="com.braintreepayments.api.dropin.DropInActivity"
            android:theme="@style/bt_drop_in_activity_theme"/>
        <activity android:name="com.braintreepayments.api.dropin.AddCardActivity"
            android:theme="@style/bt_add_card_activity_theme" />
```
